test/Microsoft.AspNetCore.Mvc.Core.Test/Infrastructure/DefaultActionDescriptorCollectionProviderTest.cs

```diff
@@ -66,55 +65,7 @@ public void ActionDescriptors_CachesValuesByDefault()
         }
 
         [Fact]
-        public void ActionDescriptors_UpdateWhenChangeTokenProviderChanges()
```

Comment:
I deleted this test because it was a strict subset of the other test that tests change tokens 😆 One test does one cycle, and another test does two.